x/mev/keeper/grpc_query.go

```diff
@@ -0,0 +1,37 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/mev/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// PendingBundles implements the Query/PendingBundles gRPC method
+func (k Keeper) PendingBundles(c context.Context, req *types.QueryPendingBundlesRequest) (*types.QueryPendingBundlesResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+	fmt.Println("Querying pending bundles for height", ctx.BlockHeight())
```

Comment:
use the SDK logger for context. 

I commented earlier about keepers being about state and so it looks like you're pulling pending bundles out of the chain here.